src/main/java/com/nkh/ECommerceShop/security/jwt/AuthEntryPointJwt.java

```diff
@@ -7,34 +7,30 @@
 import jakarta.servlet.http.HttpServletResponse;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.http.MediaType;
 import org.springframework.security.core.AuthenticationException;
 import org.springframework.security.web.AuthenticationEntryPoint;
 import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.HandlerExceptionResolver;
 
 import java.io.IOException;
 import java.util.HashMap;
 import java.util.Map;
 
 @Component
 public class AuthEntryPointJwt implements AuthenticationEntryPoint {
+  @Autowired
+  @Qualifier("handlerExceptionResolver")
```

Comment:
do you need the @Qualifier there?

Reply:
yes. it was in 2 different tutorials and I get error on removing it.